uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -7,7 +7,7 @@ Feature: GGMQ-1
     Given my device is registered as a Thing
     And my device is running Greengrass
 
-  Scenario: GGMQ-1-T1: As a customer, I can connect, subscribe, publish and receive using client application to MQTT topic
+  Scenario Outline: GGMQ-1-T1: As a customer, I can connect, subscribe, publish and receive using client application to MQTT topic
```

Comment:
Lets add parameter to scenario name GGMQ-1-T1-<mqtt-v> if it is possible

Reply:
added